doc/rtd/development/integration_tests.rst

```diff
@@ -65,6 +218,39 @@ For a minimal setup using LXD, write the following to
 
     [lxd]
 
+
+For more information on configuring pycloudlib, see the
+`pycloudlib configuration documentation`_.
+
+To specify a specific cloud to test against, first, ensure that your pycloudlib
+configuration is set up correctly. Then, modify the ``PLATFORM`` variable to be
+on of:
+
+- ``azure``: Microsoft Azure
```

Comment:
This feels like one of those lists we'll forget to update as time goes on. [integration_settings.py](https://github.com/canonical/cloud-init/blob/779dd6b009f8a568fe74e98df9803ad590b83044/tests/integration_tests/integration_settings.py#L18) already enumerates the list of valid clouds. Any way we can just point there instead?

Reply:
I was wondering about that too, but was thinking that this is probably one of the areas that will change the slowest, right? Updating docs as code changes is just kind of the pain that comes with documentation, and the code in this case will seldom ever change, so I think worrying about that is futile, no?

Reply:
I guess...to me it's already documented in the integration_settings.py file, so it feels a bit redundant here. I'll leave it up to you.